<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>783</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(104, 38, 0);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(90, 49, 19);</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Monotype Corsiva</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Список ваших файлов</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Monotype Corsiva</family>
          <pointsize>14</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Поиск</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(232, 183, 128);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="search_line">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>90</red>
              <green>49</green>
              <blue>19</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>90</red>
              <green>49</green>
              <blue>19</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>90</red>
              <green>49</green>
              <blue>19</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Shadow">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>24</red>
              <green>53</green>
              <blue>105</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>90</red>
              <green>49</green>
              <blue>19</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>90</red>
              <green>49</green>
              <blue>19</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>90</red>
              <green>49</green>
              <blue>19</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Shadow">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>24</red>
              <green>53</green>
              <blue>105</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Button">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>90</red>
              <green>49</green>
              <blue>19</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Base">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>90</red>
              <green>49</green>
              <blue>19</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Window">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>90</red>
              <green>49</green>
              <blue>19</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="Shadow">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>24</red>
              <green>53</green>
              <blue>105</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(90, 49, 19);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_search">
        <property name="font">
         <font>
          <family>Monotype Corsiva</family>
          <pointsize>14</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="text">
         <string>Найти</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_load">
        <property name="font">
         <font>
          <family>Monotype Corsiva</family>
          <pointsize>14</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(232, 183, 128);</string>
        </property>
        <property name="text">
         <string>Загрузить файл</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_read">
        <property name="font">
         <font>
          <family>Monotype Corsiva</family>
          <pointsize>14</pointsize>
          <italic>true</italic>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(232, 183, 128);</string>
        </property>
        <property name="text">
         <string>Читать выбранный файл</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_del">
        <property name="font">
         <font>
          <family>Monotype Corsiva</family>
          <pointsize>14</pointsize>
          <italic>true</italic>
          <strikeout>false</strikeout>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(232, 183, 128);</string>
        </property>
        <property name="text">
         <string>Удалить выбранный файл</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_theme">
        <property name="font">
         <font>
          <family>Monotype Corsiva</family>
          <pointsize>14</pointsize>
          <italic>true</italic>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(232, 183, 128);</string>
        </property>
        <property name="text">
         <string>Изменить тему приложения</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
